### PR TITLE
Node v12 Support

### PR DESCRIPTION
**Pull request by kumavis**
hardcoded `ethereumjs-wallet` dep is impacting node v12 support. tho, better to actually upgrade `ethereumjs-wallet` to v1

### CODE REVIEW

**Comment by kumavis** [comment]
explicit jest env dep is a workaround for Buffer instance failing `instanceof Uint8Array` check
 https://github.com/facebook/jest/issues/7780#issuecomment-669828353

**Comment by kumavis** [comment]
tested locally against node v12
id like to switch ci to github actions for easier testing across many versions of node

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "eth-rpc-errors": "^4.0.0",
     "eth-sig-util": "^3.0.0",
     "ethereumjs-util": "^6.1.0",
-    "ethereumjs-wallet": "0.6.0",
+    "ethereumjs-wallet": "^0.6.4",
```
Comment (rekmarks) [inline package.json:49]: Was this supposed to be updated to v1?

**Review by rekmarks — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kumavis) [reply package.json:49]: no, i prepared an in-range fix https://github.com/ethereumjs/ethereumjs-wallet/commits/v0.6.x
I'd be happy with a bump to v1 but out-of-scope for this

**Review by rekmarks — APPROVED** [review]
Looks

Good

to

Me!

**Comment by kumavis** [comment]
hey guysss smash that squash and merge and be sure to hit the bell to know when ive opened another pull request 🆒

**Comment by kumavis** [comment]
uh and i guess publish

**Pull request merged**